### ISSUE TITLE
Using starter template.

### ISSUE THREAD

**Issue by ShaungBhone** [tighten/jigsaw#678]
All of the starter templates are out date including docs.

**Comment by bakerkretzmar** [comment]
Not sure what you mean, please open a discussion if you have a question or an issue or PR if you think something is broken.

**Comment by sambenne** [comment]
The issue is that the composer is out of date in the starter templates so you can't install them:
```
$ vendor/bin/jigsaw init docs

> composer require tightenco/jigsaw-docs-template
./composer.json has been updated
Running composer update tightenco/jigsaw-docs-template
Loading composer repositories with package information
Updating dependencies
Lock file operations: 2 installs, 0 updates, 0 removals
  - Locking samdark/sitemap (2.4.0)
  - Locking tightenco/jigsaw-docs-template (v1.1.2)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 2 installs, 0 updates, 0 removals
  - Downloading samdark/sitemap (2.4.0)
  - Downloading tightenco/jigsaw-docs-template (v1.1.2)
  - Installing samdark/sitemap (2.4.0): Extracting archive
  - Installing tightenco/jigsaw-docs-template (v1.1.2): Extracting archive
Generating autoload files
35 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
No security vulnerability advisories found
Using version ^1.1 for tightenco/jigsaw-docs-template


> composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
- Required package "tightenco/jigsaw" is in the lock file as "v1.3.35" but that does not satisfy your constraint "^1.6".
- Required package "tightenco/jigsaw-docs-template" is not present in the lock file.
This usually happens when composer files are incorrectly merged or the composer.json file is manually edited.
Read more about correctly resolving merge conflicts https://getcomposer.org/doc/articles/resolving-merge-conflicts.md
and prefer using the "require" command over editing the composer.json file directly https://getcomposer.org/doc/03-cli.md#require-r

There was an error running the command 'composer install'
```

**Comment by bakerkretzmar** [comment]
@sambenne thanks, fixed in v1.1.3 of the docs template.
